region/tourian/main/Metroid Room 3.json

```diff
@@ -205,22 +224,25 @@
       "requires": [
         {"enemyKill": {
           "enemies": [["Metroid", "Metroid", "Metroid"]],
-          "excludedWeapons": ["Super", "Missile"]
+          "explicitWeapons": ["PowerBombPeriphery"]
         }},
-        {"metroidFrames": 272}
+        {"metroidFrames": 300}
       ],
       "clearsObstacles": ["A"]
     },
     {
       "link": [1, 2],
```

Comment:
I think i would delete this strat. It's easier to cross the room and kill them where there is a higher ceiling.

Reply:
Crossing the room positions the Metroids differently making that strat much harder / less normalized.
But I don't think that is a reason to attach `comeInNormally` to each PB kill strat for each room.  But I'd like to leave this, with a note saying to fight them on the right.